setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='simplify-debts',
+    version='1.0',
+    description='Tool to perform mutual debt simplification',
+    url='https://github.com/ric2b/Mutual_Debt_Simplification',
+    license='MIT',
+    author='',
+    author_email='',
+
+    packages=find_packages(exclude=['*.tests']),
+
+    install_requires=['graphviz==0.8'],
```

Comment:
I didn't test it but I think this is just python bindings, you still need to install graphviz through the package manager for the script to work, no? If so, leaving a message after the installation finishes would be nice.

Reply:
No. If you either install the package using pip or just by running `setup.py install`, it will install graphviz version 0.8 every time.

Reply:
I just tested it and it does fail because the python package is just a wrapper/bindings/whatever for the actual package:
`graphviz.backend.ExecutableNotFound: failed to execute ['dot', '-Tpdf', '-O', 'Initial_Mutual_Debt.gv'], make sure the Graphviz executables are on your systems' PATH`

Fixed by installing the actual package: `sudo apt install graphviz`

Reply:
I had no issue with this. I installed the application on a clean virtual environment and I was able to use without any issue, including the generation of the "debt" graph", which requires the graphviz package. How did you install the package?

Try to install using this command:
     
    pip3 install git+https://github.com/ric2b/Mutual_Debt_Simplification.git@master

Reply:
My mistake. You are right: the system package *graphviz* must be installed for the program to be able to render the graph to a PDF